bin/local-env/install-wordpress.sh

```diff
@@ -152,24 +154,28 @@ if [ "$WEBSTORIES_DEV_MODE" != $WEBSTORIES_DEV_MODE_CURRENT ]; then
   echo -e $(status_message "WEBSTORIES_DEV_MODE: $WEBSTORIES_DEV_MODE_RESULT...")
 fi
 
+MEDIA_TRASH_CURRENT=!MEDIA_TRASH;
+if [ "$(wp config has --type=constant MEDIA_TRASH)" ]; then
+  $MEDIA_TRASH_CURRENT=$(wp config get --type=constant --format=json MEDIA_TRASH | tr -d '\r')
+fi
+if [ "$MEDIA_TRASH" != $MEDIA_TRASH_CURRENT ]; then
+  wp config set MEDIA_TRASH $MEDIA_TRASH --raw --type=constant --quiet
+  MEDIA_TRASH_RESULT=$(wp config get --type=constant --format=json MEDIA_TRASH | tr -d '\r')
+  echo -e $(status_message "MEDIA_TRASH: $MEDIA_TRASH_RESULT...")
+fi
+
 # Let's make sure we have some media in the media library to work with.
 echo -e $(status_message "Import default set of media assets...")
 # TODO: use glob pattern to import items. See https://developer.wordpress.org/cli/commands/media/import/.
 
 # Since MOV files are not allowed in the editor, only the WEBM one needs a poster.
 wp media import /var/www/html/wp-content/e2e-assets/small-video.mov --quiet
 
-WEBM_VIDEO_ID=$(wp media import /var/www/html/wp-content/e2e-assets/small-video.webm --porcelain)
-WEBM_VIDEO_POSTER_ID=$(wp media import /var/www/html/wp-content/e2e-assets/small-video-poster.jpg --post_id=$WEBM_VIDEO_ID --featured_image --porcelain)
-
-# So the poster is marked as such and hidden in the editor.
-wp post term add $WEBM_VIDEO_POSTER_ID web_story_media_source "poster-generation" --quiet
-wp post meta add $WEBM_VIDEO_ID web_stories_poster_id $WEBM_VIDEO_POSTER_ID --quiet
-
```

Comment:
So we don't want any video available in the editor by default, only images?

Reply:
Honestly don't see the point. We don't populate other test data like example stories, why example assets.

Reply:
I always thought it would be handy to have some example assets ready to quickly assemble a story in tests. Like we do in karma tests. So you don't have to upload an image/video every time. But I suppose so far we haven't really made use of that.

Reply:
I put back the video for now, I think you are right, I think there is value in there being in local dev. I have, however, removed the svg and mov, as they are only used in the e2e tests and are now covered by this change.

Reply:
You said you put the video back, but it still shows as deleted here?

Reply:
It was back locally but for some reason it didn't push. It must have been some kind of revert locally before it was committed. Surprised to find this. It is back now.